packages/github-actions/actions/automerge-released-trunk/README.md

```diff
@@ -0,0 +1,29 @@
+# Auto merge a released trunk
+
+This action provides the following functionality for GitHub Actions users:
+
+- Automatically merge trunk to develop after a release done with the `woocommerce/grow/prepare-extension-release`
+
+## Usage
+
+See [action.yml](action.yml)
+
+#### Basic:
+
+```yaml
+name: Auto merge a released trunk
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - trunk
+
+jobs:
+  automerge_trunk:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Considering this action requires this step to initial the repo and some `git` config, maybe it would be a bit better to move this step into this action instead of asking users to add it to their workflow.

![image](https://github.com/woocommerce/grow/assets/17420811/e9ff893d-10dc-4be0-a099-67ddd6e55e7c)